### ISSUE TITLE
Installation fails on osx+brew+node-0.12

### ISSUE THREAD

**Issue by orefalo** [iron-meteor/iron-cli#95]
$> npm install -g iron-meteor                                         09:37:20

> fibers@1.0.1 install /usr/local/lib/node_modules/iron-meteor/node_modules/fibers
> node ./build.js

child_process: customFds option is deprecated, use stdio instead.
child_process: customFds option is deprecated, use stdio instead.
  CXX(target) Release/obj.target/fibers/src/fibers.o
../src/fibers.cc:222:34: error: unknown type name 'Arguments'; did you mean
      'v8::internal::Arguments'?
                static Handle<Value> New(const Arguments& args) {
                                               ^~~~~~~~~
                                               v8::internal::Arguments
/Users/orefalo/.node-gyp/0.12.0/deps/v8/include/v8.h:127:7: note:
      'v8::internal::Arguments' declared here
class Arguments;
      ^
../src/fibers.cc:241:34: error: unknown type name 'Arguments'; did you mean
      'v8::internal::Arguments'?
                static Handle<Value> Run(const Arguments& args) {
                                               ^~~~~~~~~
                                               v8::internal::Arguments
/Users/orefalo/.node-gyp/0.12.0/deps/v8/include/v8.h:127:7: note:
      'v8::internal::Arguments' declared here
class Arguments;
      ^
../src/fibers.cc:279:40: error: unknown type name 'Arguments'; did you mean
      'v8::internal::Arguments'?
                static Handle<Value> ThrowInto(const Arguments& args) {
                                                     ^~~~~~~~~
                                                     v8::internal::Arguments
/Users/orefalo/.node-gyp/0.12.0/deps/v8/include/v8.h:127:7: note:
      'v8::internal::Arguments' declared here
class Arguments;
      ^
../src/fibers.cc:300:36: error: unknown type name 'Arguments'; did you mean
      'v8::internal::Arguments'?
                static Handle<Value> Reset(const Arguments& args) {
                                                 ^~~~~~~~~
                                                 v8::internal::Arguments
/Users/orefalo/.node-gyp/0.12.0/deps/v8/include/v8.h:127:7: note:
      'v8::internal::Arguments' declared here
class Arguments;
      ^
../src/fibers.cc:468:37: error: unknown type name 'Arguments'; did you mean
      'v8::internal::Arguments'?
                static Handle<Value> Yield_(const Arguments& args) {
                                                  ^~~~~~~~~
                                                  v8::internal::Arguments
/Users/orefalo/.node-gyp/0.12.0/deps/v8/include/v8.h:127:7: note:
      'v8::internal::Arguments' declared here
class Arguments;
      ^
../src/fibers.cc:510:65: error: unknown type name 'AccessorInfo'
  ...static Handle<Value> GetStarted(Local<String> property, const AccessorIn...
                                                                   ^
../src/fibers.cc:518:65: error: unknown type name 'AccessorInfo'
  ...static Handle<Value> GetCurrent(Local<String> property, const AccessorIn...
                                                                   ^
../src/fibers.cc:529:66: error: unknown type name 'AccessorInfo'
  ...static Handle<Value> GetPoolSize(Local<String> property, const AccessorI...
                                                                    ^
../src/fibers.cc:533:77: error: unknown type name 'AccessorInfo'
  ...SetPoolSize(Local<String> property, Local<Value> value, const AccessorIn...
                                                                   ^
../src/fibers.cc:540:71: error: unknown type name 'AccessorInfo'
  ...Handle<Value> GetFibersCreated(Local<String> property, const AccessorInf...
                                                                  ^
../src/fibers.cc:202:24: error: no matching constructor for initialization of
      'String::Utf8Value'
                                        String::Utf8Value stack(fatal_stack);
                                                          ^     ~~~~~~~~~~~
/Users/orefalo/.node-gyp/0.12.0/deps/v8/include/v8.h:1889:14: note: candidate
      constructor not viable: no known conversion from 'Persistent<v8::Value>'
      to 'Handle<v8::Value>' for 1st argument
    explicit Utf8Value(Handle<v8::Value> obj);
             ^
/Users/orefalo/.node-gyp/0.12.0/deps/v8/include/v8.h:1899:5: note: candidate
      constructor not viable: no known conversion from 'Persistent<v8::Value>'
      to 'const v8::String::Utf8Value' for 1st argument
    Utf8Value(const Utf8Value&);
    ^
../src/fibers.cc:210:18: error: no member named 'Dispose' in
      'v8::Persistent<v8::Value, v8::NonCopyablePersistentTraits<v8::Value> >'
                                        fatal_stack.Dispose();
                                        ~~~~~~~~~~~ ^
../src/fibers.cc:213:18: error: no member named 'Dispose' in
      'v8::Persistent<v8::Value, v8::NonCopyablePersistentTraits<v8::Value> >'
                                that.yielded.Dispose();
                                ~~~~~~~~~~~~ ^
../src/fibers.cc:223:12: error: member access into incomplete type 'const
      v8::internal::Arguments'
                        if (args.Length() != 1) {
                                ^
/Users/orefalo/.node-gyp/0.12.0/deps/v8/include/v8.h:127:7: note: forward
      declaration of 'v8::internal::Arguments'
class Arguments;
      ^
../src/fibers.cc:224:5: error: no member named 'New' in 'v8::String'; did you
      mean simply 'New'?
  ...THROW(Exception::TypeError, "Fiber expects 1 argument");
     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../src/fibers.cc:9:45: note: expanded from macro 'THROW'
#define THROW(x, m) return ThrowException(x(String::New(m)))
                                            ^~~~~~~~
../src/fibers.cc:222:24: note: 'New' declared here
                static Handle<Value> New(const Arguments& args) {
                                     ^
../src/fibers.cc:224:33: error: reference to type
      'const v8::internal::Arguments' could not bind to an lvalue of type
      'const char [25]'
  ...THROW(Exception::TypeError, "Fiber expects 1 argument");
                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~
../src/fibers.cc:9:57: note: expanded from macro 'THROW'
#define THROW(x, m) return ThrowException(x(String::New(m)))
                                                        ^
../src/fibers.cc:222:45: note: passing argument to parameter 'args' here
                static Handle<Value> New(const Arguments& args) {
                                                          ^
../src/fibers.cc:225:20: error: type 'const v8::internal::Arguments' does not
      provide a subscript operator
                        } else if (!args[0]->IsFunction()) {
                                    ~~~~^~
../src/fibers.cc:226:5: error: no member named 'New' in 'v8::String'; did you
      mean simply 'New'?
  ...THROW(Exception::TypeError, "Fiber expects a function");
     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../src/fibers.cc:9:45: note: expanded from macro 'THROW'
#define THROW(x, m) return ThrowException(x(String::New(m)))
                                            ^~~~~~~~
../src/fibers.cc:222:24: note: 'New' declared here
                static Handle<Value> New(const Arguments& args) {
                                     ^
../src/fibers.cc:226:33: error: reference to type
      'const v8::internal::Arguments' could not bind to an lvalue of type
      'const char [25]'
  ...THROW(Exception::TypeError, "Fiber expects a function");
                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~
../src/fibers.cc:9:57: note: expanded from macro 'THROW'
#define THROW(x, m) return ThrowException(x(String::New(m)))
                                                        ^
../src/fibers.cc:222:45: note: passing argument to parameter 'args' here
                static Handle<Value> New(const Arguments& args) {
                                                          ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
make: *** [Release/obj.target/fibers/src/fibers.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:267:23)
gyp ERR! stack     at ChildProcess.emit (events.js:110:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1067:12)
gyp ERR! System Darwin 14.3.0
gyp ERR! command "node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /usr/local/lib/node_modules/iron-meteor/node_modules/fibers
gyp ERR! node -v v0.12.0
gyp ERR! node-gyp -v v1.0.2
gyp ERR! not ok
Build failed
npm ERR! Darwin 14.3.0
npm ERR! argv "node" "/usr/local/bin/npm" "install" "-g" "iron-meteor"
npm ERR! node v0.12.0
npm ERR! npm  v2.5.1
npm ERR! code ELIFECYCLE

npm ERR! fibers@1.0.1 install: `node ./build.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the fibers@1.0.1 install script 'node ./build.js'.
npm ERR! This is most likely a problem with the fibers package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node ./build.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls fibers
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/xxx/npm-debug.log

**Comment by orefalo** [comment]
found the solution, you need to update fibers dependency to 1.0.5

**Comment by orefalo** [comment]
and here is the pull req https://github.com/iron-meteor/iron-cli/pull/96

in the meantime, use this `npm install -g  orefalo/iron-cli`

**Comment by cmather** [comment]
Yeah probably a fibers version issue. I was trying to stay with meteor's version of nodejs. Best solution would be to require a nodejs version and use a command tool like 'n' (search npm packages) to have multiple versions of node installed.

**Comment by nannigan** [comment]
Also on OSX and seem to have had installation thwarted by fibers also, even using orefalo's alternative:
1652 error fibers@1.0.5 install: `node ./build.js`
1652 error Exit status 127
1653 error Failed at the fibers@1.0.5 install script.
1653 error This is most likely a problem with the fibers package,
1653 error not with npm itself.
1653 error Tell the author that this fails on your system:
1653 error     node ./build.js
1653 error You can get their info via:
1653 error     npm owner ls fibers
1653 error There is likely additional logging output above.
1654 error System Darwin 13.4.0
...
1657 error node -v v0.10.28
1658 error npm -v 1.4.9
1659 error code ELIFECYCLE
1660 verbose exit [ 1, true ]

**Comment by cmather** [comment]
For the mac/linux users, try using n to use version 0.10.29 of node. Actually, 'n' is pretty useful because you can have multiple versions of node installed at once. Do this:

```
sudo npm install -g n
```

Then you can install 0.10.29 and use it like this

```
n 0.10.29
```

See if the iron-cli tool works with 0.10.29. I'll update it soon to work with the latest version of node.

**Comment by nannigan** [comment]
getting the same errors after installing node version 0.10.29 
(btw thanks for info re 'n'.)

**Comment by cmather** [comment]
I wonder if you have to remove it first. Maybe npm uninstall iron-meteor and then try again. Maybe it's already compiled against the wrong version and isn't going through the recompile again. The version of fibers should be 1.0.1.

**Comment by nannigan** [comment]
On Mar 4, 2015, at 6:58 PM, Chris Mather <notifications@github.com> wrote:

> I wonder if you have to remove it first. Maybe npm uninstall iron-meteor and then try again. Maybe it's already compiled against the wrong version and isn't going through the recompile again.
> 
tried uninstall got
npm WARN uninstall not installed in /Users/ashby/node_modules: "iron-meteor"
> The version of fibers should be 1.0.1.
> 
it is referenced at that version in the series of error messages:
npm ERR! fibers@1.0.1 install: `node ./build.js`
npm ERR! Exit status 127
npm ERR! 
npm ERR! Failed at the fibers@1.0.1 install script.

**Comment by cmather** [comment]
Are you able to link from source?

```bash
> git clone https://github.com/iron-meteor/iron-cli.git
> cd iron-cli
> npm install
> sudo npm link
```

**Comment by nannigan** [comment]
On Mar 4, 2015, at 7:11 PM, Chris Mather <notifications@github.com> wrote:

> Are you able to link from source?
> 
> > git clone https://github.com/iron-meteor/iron-cli.git
> > cd iron-cli
> > npm install
> > sudo npm link
at this point (after sudo npm link) I get
npm WARN package.json cli-table@0.3.0 No repository field.
/usr/local/bin/iron -> /usr/local/lib/node_modules/iron-meteor/bin/iron
/usr/local/lib/node_modules/iron-meteor -> /Users/ashby/Sites/meteor/iron-cli

I’m swimming a little deep here, not sure what was to be expected…

thanks for your help!

**Comment by cmather** [comment]
Warnings are okay. Did it install? Can you type 'iron' in the command line? Swimming deep makes you stronger :).

**Comment by nannigan** [comment]
On Mar 4, 2015, at 7:53 PM, Chris Mather <notifications@github.com> wrote:

> Warnings are okay. Did it install? Can you type 'iron' in the command line?
> 
command not found …
I tried it from the newly cloned iron-cli directory, a sibling project directory  and from the parent directory
> Swimming deep makes you stronger :).
> 
;^)

**Comment by cmather** [comment]
blasted. are there actual errors when you npm link? or just warnings. Does the 'iron' executable exist in the /usr/local/bin/ path? Also, what version of node are you on so I know for future reference? see what's in your PATH env variable and make sure /usr/local/bin is in it.

```bash
> ls -al /usr/local/bin/iron
```
Anything at that location?

```bash
> echo "$PATH"
```
What's in your path?

**Comment by cmather** [comment]
Oops disregard the node version question. I guess we already talked about that.

**Comment by bojicas** [comment]
@nannigan 

Sorry to pop uninvited in the discussion :)

I got the initial reported problem, too (it seems a fiber 1.0.1 - latest node incompatibility), and the solution was indeed to install ```n``` and the appropriate version ```0.10.29``` before reinstalling ```iron-meteor```. Note also that I am on a Mac (OS X Yosemite) and that the original node was installed via ```brew``` (no ```sudo``` prefixed commands in my case).

Let me reproduce the sequence of the steps that worked for me below:

```
npm uninstall iron-meteor
npm install -g n
n 0.10.29
node -v
npm -v
npm install -g iron-meteor
```

```node -v``` step confirmed the node version ```v0.10.29``` and ```npm -v``` step the npm version ```1.4.14```.

Hope that this will work for everybody else with same issue, too.

**Comment by bojicas** [comment]
One more thing to add, although ```fibers``` dependency upgrade to version ```1.0.5``` looks like the easiest trick, my understanding is that using ```node``` locked at version ```0.10.29``` and ```fibers``` at ```1.0.1``` is desired in order to keep the version of these packages aligned to those used my ```meteor``` itself.

**Comment by cmather** [comment]
Thanks for the additional perspective! And true statement about the nodejs version. Although it's not really necessary because meteor uses its own node binary. So it's more that v0.10.29 happened to be what I was using. I will try to update and push an update this week to the latest version of node.

**Comment by nannigan** [comment]
I did try the steps that worked for Silvlu, but had the same failure as before.
I am on OS10.9.4 and not using brew, though I do have version .9.5 installed it seems...



On Mar 4, 2015, at 8:41 PM, Chris Mather <notifications@github.com> wrote:

> blasted. are there actual errors when you npm link? or just warnings. Does the 'iron' executable exist in the /usr/local/bin/ path?
> 
npm WARN package.json cli-table@0.3.0 No repository field.
/usr/local/bin/iron -> /usr/local/lib/node_modules/iron-meteor/bin/iron
/usr/local/lib/node_modules/iron-meteor -> /Users/ashby/Sites/meteor/iron-cli
> Also, what version of node are you on so I know for future reference? see what's in your PATH env variable and make sure /usr/local/bin is in it.
> 
> > ls -al /usr/local/bin/iron
lrwxr-xr-x  1 root  wheel  40 Mar  4 23:15 /usr/local/bin/iron -> ../lib/node_modules/iron-meteor/bin/iron
> 
> > echo "$PATH”
/usr/local/heroku/bin:/usr/local/rvm/gems/ruby-1.9.3-p392/bin:/usr/local/rvm/gems/ruby-1.9.3-p392@global/bin:/usr/local/rvm/rubies/ruby-1.9.3-p392/bin:/usr/local/rvm/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin:/opt/sm/bin:/opt/sm/pkg/active/bin:/opt/sm/pkg/active/sbin


hope this provides something useful!

**Comment by Siyfion** [comment]
I think this is probably the same issue, but I too am getting issues when attempting to install on OSX:

```
$ npm install -g iron-meteor
 
> fibers@1.0.1 install /usr/local/lib/node_modules/iron-meteor/node_modules/fibers
> node ./build.js

child_process: customFds option is deprecated, use stdio instead.
child_process: customFds option is deprecated, use stdio instead.
  CXX(target) Release/obj.target/fibers/src/fibers.o
../src/fibers.cc:222:34: error: unknown type name 'Arguments'; did you mean
      'v8::internal::Arguments'?
                static Handle<Value> New(const Arguments& args) {
                                               ^~~~~~~~~
                                               v8::internal::Arguments
/Users/siyfion/.node-gyp/0.12.0/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared
      here
class Arguments;
      ^
../src/fibers.cc:241:34: error: unknown type name 'Arguments'; did you mean
      'v8::internal::Arguments'?
                static Handle<Value> Run(const Arguments& args) {
                                               ^~~~~~~~~
                                               v8::internal::Arguments
/Users/siyfion/.node-gyp/0.12.0/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared
      here
class Arguments;
      ^
../src/fibers.cc:279:40: error: unknown type name 'Arguments'; did you mean
      'v8::internal::Arguments'?
                static Handle<Value> ThrowInto(const Arguments& args) {
                                                     ^~~~~~~~~
                                                     v8::internal::Arguments
/Users/siyfion/.node-gyp/0.12.0/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared
      here
class Arguments;
      ^
../src/fibers.cc:300:36: error: unknown type name 'Arguments'; did you mean
      'v8::internal::Arguments'?
                static Handle<Value> Reset(const Arguments& args) {
                                                 ^~~~~~~~~
                                                 v8::internal::Arguments
/Users/siyfion/.node-gyp/0.12.0/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared
      here
class Arguments;
      ^
../src/fibers.cc:468:37: error: unknown type name 'Arguments'; did you mean
      'v8::internal::Arguments'?
                static Handle<Value> Yield_(const Arguments& args) {
                                                  ^~~~~~~~~
                                                  v8::internal::Arguments
/Users/siyfion/.node-gyp/0.12.0/deps/v8/include/v8.h:127:7: note: 'v8::internal::Arguments' declared
      here
class Arguments;
      ^
../src/fibers.cc:510:65: error: unknown type name 'AccessorInfo'
                static Handle<Value> GetStarted(Local<String> property, const AccessorInfo& info) {
                                                                              ^
../src/fibers.cc:518:65: error: unknown type name 'AccessorInfo'
                static Handle<Value> GetCurrent(Local<String> property, const AccessorInfo& info) {
                                                                              ^
../src/fibers.cc:529:66: error: unknown type name 'AccessorInfo'
                static Handle<Value> GetPoolSize(Local<String> property, const AccessorInfo& info) {
                                                                               ^
../src/fibers.cc:533:77: error: unknown type name 'AccessorInfo'
  ...static void SetPoolSize(Local<String> property, Local<Value> value, const AccessorInfo& info) {
                                                                               ^
../src/fibers.cc:540:71: error: unknown type name 'AccessorInfo'
                static Handle<Value> GetFibersCreated(Local<String> property, const Accessor...
                                                                                    ^
../src/fibers.cc:202:24: error: no matching constructor for initialization of 'String::Utf8Value'
                                        String::Utf8Value stack(fatal_stack);
                                                          ^     ~~~~~~~~~~~
/Users/siyfion/.node-gyp/0.12.0/deps/v8/include/v8.h:1889:14: note: candidate constructor not
      viable: no known conversion from 'Persistent<v8::Value>' to 'Handle<v8::Value>' for 1st
      argument
    explicit Utf8Value(Handle<v8::Value> obj);
             ^
/Users/siyfion/.node-gyp/0.12.0/deps/v8/include/v8.h:1899:5: note: candidate constructor not viable:
      no known conversion from 'Persistent<v8::Value>' to 'const v8::String::Utf8Value' for 1st
      argument
    Utf8Value(const Utf8Value&);
    ^
../src/fibers.cc:210:18: error: no member named 'Dispose' in 'v8::Persistent<v8::Value,
      v8::NonCopyablePersistentTraits<v8::Value> >'
                                        fatal_stack.Dispose();
                                        ~~~~~~~~~~~ ^
../src/fibers.cc:213:18: error: no member named 'Dispose' in 'v8::Persistent<v8::Value,
      v8::NonCopyablePersistentTraits<v8::Value> >'
                                that.yielded.Dispose();
                                ~~~~~~~~~~~~ ^
../src/fibers.cc:223:12: error: member access into incomplete type 'const v8::internal::Arguments'
                        if (args.Length() != 1) {
                                ^
/Users/siyfion/.node-gyp/0.12.0/deps/v8/include/v8.h:127:7: note: forward declaration of
      'v8::internal::Arguments'
class Arguments;
      ^
../src/fibers.cc:224:5: error: no member named 'New' in 'v8::String'; did you mean simply 'New'?
                                THROW(Exception::TypeError, "Fiber expects 1 argument");
                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../src/fibers.cc:9:45: note: expanded from macro 'THROW'
#define THROW(x, m) return ThrowException(x(String::New(m)))
                                            ^~~~~~~~
../src/fibers.cc:222:24: note: 'New' declared here
                static Handle<Value> New(const Arguments& args) {
                                     ^
../src/fibers.cc:224:33: error: reference to type 'const v8::internal::Arguments' could not bind to
      an lvalue of type 'const char [25]'
                                THROW(Exception::TypeError, "Fiber expects 1 argument");
                                                            ^~~~~~~~~~~~~~~~~~~~~~~~~~
../src/fibers.cc:9:57: note: expanded from macro 'THROW'
#define THROW(x, m) return ThrowException(x(String::New(m)))
                                                        ^
../src/fibers.cc:222:45: note: passing argument to parameter 'args' here
                static Handle<Value> New(const Arguments& args) {
                                                          ^
../src/fibers.cc:225:20: error: type 'const v8::internal::Arguments' does not provide a subscript
      operator
                        } else if (!args[0]->IsFunction()) {
                                    ~~~~^~
../src/fibers.cc:226:5: error: no member named 'New' in 'v8::String'; did you mean simply 'New'?
                                THROW(Exception::TypeError, "Fiber expects a function");
                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../src/fibers.cc:9:45: note: expanded from macro 'THROW'
#define THROW(x, m) return ThrowException(x(String::New(m)))
                                            ^~~~~~~~
../src/fibers.cc:222:24: note: 'New' declared here
                static Handle<Value> New(const Arguments& args) {
                                     ^
../src/fibers.cc:226:33: error: reference to type 'const v8::internal::Arguments' could not bind to
      an lvalue of type 'const char [25]'
                                THROW(Exception::TypeError, "Fiber expects a function");
                                                            ^~~~~~~~~~~~~~~~~~~~~~~~~~
../src/fibers.cc:9:57: note: expanded from macro 'THROW'
#define THROW(x, m) return ThrowException(x(String::New(m)))
                                                        ^
../src/fibers.cc:222:45: note: passing argument to parameter 'args' here
                static Handle<Value> New(const Arguments& args) {
                                                          ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
make: *** [Release/obj.target/fibers/src/fibers.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:267:23)
gyp ERR! stack     at ChildProcess.emit (events.js:110:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1067:12)
gyp ERR! System Darwin 14.1.0
gyp ERR! command "node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /usr/local/lib/node_modules/iron-meteor/node_modules/fibers
gyp ERR! node -v v0.12.0
gyp ERR! node-gyp -v v1.0.2
gyp ERR! not ok 
Build failed
npm ERR! Darwin 14.1.0
npm ERR! argv "node" "/usr/local/bin/npm" "install" "-g" "iron-meteor"
npm ERR! node v0.12.0
npm ERR! npm  v2.6.0
npm ERR! code ELIFECYCLE

npm ERR! fibers@1.0.1 install: `node ./build.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the fibers@1.0.1 install script 'node ./build.js'.
npm ERR! This is most likely a problem with the fibers package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node ./build.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls fibers
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/siyfion/Git/LabelLogicLive/npm-debug.log
```

**Comment by corv89** [comment]
The node process is using massive amounts of CPU now when running meteor via iron since fibers is at 1.0.5.

**Comment by orefalo** [comment]
Yep, fixed my install as follows:

reinstalled node using the native pkg installer from http://nodejs.org/dist/v0.10.36/
  note the version of node here, 0.10 which is the most stable production ready version.

From there everything runs as expected.
Oli

**Comment by corv89** [comment]
@orefalo the pkg for v0.10.36 did the trick, thanks.

**Comment by cmather** [comment]
This should be fixed on next release with fibers at 1.0.5

**Comment by kenorb** [comment]
This seems to worked:

```
npm install -g n
n 0.10.44
node --version
npm install -g meteor
```
